derive_client/data_types/enums.py

```diff
@@ -25,7 +25,7 @@ class TxStatus(IntEnum):
 class ChainID(IntEnum):
     ETH = 1
     OPTIMISM = 10
-    LYRA = 957
+    DERIVE = LYRA = 957
```

Comment:
this way backwards compatible (mostly) and also clarifies to reader that these are in fact the same. We might want to also mention this in the repo README.md